Document/0x05b-Basic-Security_Testing.md

```diff
@@ -370,30 +370,18 @@ More infomation on using the Objection REPL can be found on the [Objection Wiki]
 
 ###### Installing radare2
 
-Please refer to [radare2's official installation instructions](https://github.com/radare/radare2/blob/master/README.md "radare2 installation instructions"). There are also installation instructions If you need to [install radare2 on Android](https://github.com/radare/radare2/blob/master/doc/android.md "Install radare2 on Android").
+Please refer to [radare2's official installation instructions](https://github.com/radare/radare2/blob/master/README.md "radare2 installation instructions"). There are also installation instructions if you need to [install radare2 on Android](https://github.com/radare/radare2/blob/master/doc/android.md "Install radare2 on Android").
 
 ###### Using radare2
 
 You can use `rafind2` to read strings directly from the encoded AndroidManifest.xml. Use this to list permissions, activities, providers, services, receivers and other information stored in AndroidManifest.xml
 
 ```shell
-$ rafind2 -ZS permission AndroidManifest.xml
-```
-
-```shell
-$ rafind2 -ZS activity AndroidManifest.xml
-```
-
-```shell
-$ rafind2 -ZS provider AndroidManifest.xml
-```
-
-```shell
-$ rafind2 -ZS service AndroidManifest.xml
-```
-
-```shell
-$ rafind2 -ZS receiver AndroidManifest.xml
+$ rafind2 -ZS permission AndroidManifest.xml # Permissions
```

Comment:
Put them before the command. Looks overkill now, but it's consistent :)

Reply:
Fixed.